### PR TITLE
Adds prerequisite for disabling AWS VPC CNI network policy

### PR DESCRIPTION
**Pull request by ctauchen**
In the spirit of #1612. Also removes outdated note from OSS.

### CODE REVIEW

**Comment by ctauchen** [comment]
@tomastigera Figured it was easier to put this in a PR than ask you to refactor. Simplified and changed to prerequisite step for the Calico-policy-only procedure.

File: calico/getting-started/kubernetes/managed-public-cloud/eks.mdx [hunk 1]
```diff
@@ -44,6 +34,9 @@ When using the Amazon VPC CNI plugin, {{prodname}} does not support enforcement
 
 :::
 
+***Prerequisites***
+
+* You [disabled network policy for the AWS VPC CNI](https://docs.aws.amazon.com/eks/latest/userguide/network-policy-disable.html).
```
Comment (tomastigera) [inline calico/getting-started/kubernetes/managed-public-cloud/eks.mdx:39]:
```suggestion
* You must [disable network policy for the AWS VPC CNI](https://docs.aws.amazon.com/eks/latest/userguide/network-policy-disable.html). 
```

Comment (ctauchen) [reply calico/getting-started/kubernetes/managed-public-cloud/eks.mdx:39]: Thanks, but I'm going to stick with this format. The idea of 'must' is already there, given that this is a prerequisite section.

**Review by ctauchen — COMMENTED** [review]
Great, I've added the same content to the CE docs. Happy for this to get merged @tomastigera ?

**Pull request merged**